src/main/java/Drink.java

```diff
@@ -0,0 +1,9 @@
+public class Drink {
+    String name;
+    double price;
+    boolean alcohol;
+
+    Ingredient typeOfalcohol;
+    Ingredient typeOfjuice;
+    Ingredient flavoredLiqueur;
```

Comment:
Sugeruję zmienić nazwy tych pól. na coś bardziej ogólnego: ingredient1, ingredient2 itd. Chodzi o to, że pod `typeOfalcohol` można wstawić dowolny składnik (np. sok), a w takim przypadku nazwa byłaby myląca.